lib/grand_total_nodes.js

```diff
@@ -0,0 +1,11 @@
+class GrandTotalNodes {
+    constructor(tables) {
+        this.goal = tables.find('#goal-calories')
+        this.total = tables.find('#total-calories')
+        this.remaining = tables.find('#remaining-calories')
+        this.tableTotals = tables.find('.total-calories')
+    }
+
```

Comment:
Nice. This seems like a really good way to break out more functionality into separate pieces, as well as add some efficiency and readability.